ThirdPartyNotices.txt

```diff
@@ -10,6 +10,7 @@ This project incorporates components from the projects listed below. The origina
 3.	lbugnion/mvvmlight commit 4cbf77c (https://github.com/lbugnion/mvvmlight), from which some APIs from the `Microsoft.Toolkit.Mvvm` package take inspiration from.
 4.  PrivateObject/PrivateType (https://github.com/microsoft/testfx/tree/664ac7c2ac9dbfbee9d2a0ef560cfd72449dfe34/src/TestFramework/Extension.Desktop), included in UnitTests.
 5.  QuinnDamerell/UniversalMarkdown (https://github.com/QuinnDamerell/UniversalMarkdown) contributed by Quinn Damerell and Paul Bartrum for the MarkdownTextBlock control, relicensed to this .NET Foundation project under the MIT license upon contribution in https://github.com/windows-toolkit/WindowsCommunityToolkit/pull/772.
+6.  qmatteoq/DesktopBridgeHelpers commit e278153 (https://github.com/qmatteoq/DesktopBridgeHelpers), using in DesktopNotificationManagerCompat.cs and DesktopBridgeHelpers.cs to identify if running with identity.
```

Comment:
@andrewleader assuming you borrowed some of the code to use within the toolkit since I don't see a NuGet reference. Appreciate the appropriate attribution here.

However, since @qmatteoq is a fellow MS employee, it may just be easier to have him also [sign our CLA](https://cla.dotnetfoundation.org/windows-toolkit/WindowsCommunityToolkit?pullRequest=3457) and sign-off with a comment that he's good relicensing this to this project and the .NET Foundation as part of this PR. Then we wouldn't need to conflate the licenses here with extra complexity duplicating the MIT license.

Reply:
Good point, @qmatteoq can you [sign the CLA](https://cla.dotnetfoundation.org/windows-toolkit/WindowsCommunityToolkit?pullRequest=3457) and sign-off with a comment that you're good relicensing this to this project and the .NET Foundation as part of this PR? The borrowed code is here: https://github.com/windows-toolkit/WindowsCommunityToolkit/blob/aleader/notifications-registry/Microsoft.Toolkit.Uwp.Notifications/Toasts/Compat/Desktop/DesktopBridgeHelpers.cs

Reply:
I truly apologize for the delay, but I was on sick leave and I missed most of the notifications. I've signed the CLA and, of course, I'm perfectly fine in relicensing my code to this project.

Reply:
Thanks @qmatteoq, much appreciated! Sorry to hear about being out sick, hope you are feeling better now!

@andrewleader I think this means we can just make this line in the same format as the one above it pointing to Matteo's comment in this PR and then remove the extra license copy at the bottom. Happy to clean the file up for you if you'd like too, let me know.

Reply:
I just made those changes @michael-hawker! Thanks @qmatteoq, glad you were taking time off to feel better!